lib/like-pattern-concordance.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const concordance = require('concordance');
+const isPlainObject = require('is-plain-object');
+
+function pickLikePatternPaths(actual, likePattern, visited = new WeakMap()) {
+	if (visited.has(likePattern)) {
+		return visited.get(likePattern);
+	}
+
+	if (isPlainObject(likePattern) && isPlainObject(actual)) {
+		const actualPicked = {};
+
+		visited.set(likePattern, actualPicked);
+
+		for (const key of Object.keys(likePattern)) {
+			if (key in actual) {
```

Comment:
I think this should use `Reflect.has(actual, key)`, depending on the answer to the above.

Reply:
Is there a difference between `Reflect.has` and `in`-operator? I don't think there is, so this does not matter (unless this is an issue of code style or something).

Reply:
`in` walks the prototype chain, `Reflect.has()` does not.

Reply:
```
> Reflect.has({}, 'toString')
true
```

Did you mean `Object.hasOwnProperty`?

Reply:
Yes, but `Reflect.has()` is the better version of that.